.gitlab-ci.yml

```diff
@@ -39,7 +39,7 @@ before_script:
     OS_VER: '3.5'
     TAG: '1.10.0'
     VER: '1.10.0'
-    DVER: '1.10.3'
+    DVER: '1.x'
```

Comment:
Is it a kind of docker file to use, which works like a template specialized by providing `--build-arg` options? If so, I understand the change and it looks okay. But the variable should have more meaningful name. Like, maybe, DOCKERFILE_NAME_SUFFIX.

Reply:
Right, ok, naming changed.